extensions/amp-imgur/0.1/amp-imgur.js

```diff
@@ -79,10 +79,10 @@ export class AmpImgur extends AMP.BaseElement {
     iframe.setAttribute('frameborder', '0');
     iframe.setAttribute('allowfullscreen', 'true');
 
-    iframe.src =
-      'https://imgur.com/' +
-      encodeURIComponent(this.imgurid_) +
-      '/embed?pub=true';
+    const sanitizedID = startsWith(this.imgurid_, 'a/')
+      ? 'a/' + encodeURIComponent(this.imgurid_.replace('a/', ''))
```

Comment:
You should do

```javascript
const sanitizedID = startsWith(this.imgurid_, 'a/')
  ? 'a/' + encodeURIComponent(this.imgurid_.replace(/^a\//, ''))
```

since you are checking for (and prepending 'a'), but your replace will replace it anywhere in the string. While it isn't clear that a slash is a valid character in an id, you should make your code robust to this.